src/_data/schedule/day_two.yml

```diff
@@ -0,0 +1,54 @@
+- start_at: "8:00"
+  end_at: "9:00"
+  title: "Conference Door Opens"
+- start_at: "9:00"
+  end_at: "9:10"
+  title: "Welcome Address"
+- start_at: "9:10"
+  end_at: "9:55"
+  title: "Ruby & JVM: A (JRuby) Love Story"
+  speaker_id: yarden-laifenfeld
+- start_at: "10:00"
+  end_at: "10:30"
+  title: "On making your Rails App more transparent"
+  speaker_id: stephan-eberle
+- start_at: "10:30"
+  end_at: "10:50"
+  title: "Tea Break"
+- start_at: "10:50"
+  end_at: "11:20"
+  title: "GitOps: The Single source of truth"
+  speaker_id: thatthep-vorrasing
+- start_at: "11:25"
+  end_at: "11:55"
+  title: "Dealing With A Project's Complexity In A Changing Environment"
+  speaker_id: anita-jaszewska
+- start_at: "12:00"
+  end_at: "12:30"
+  title: "Dissecting Rails - a different approach to learning Rails"
+  speaker_id: rtdp
+- start_at: "12:30"
+  end_at: "13:50"
+  title: "Buffet Lunch"
```

Comment:
probably useful if we can add subtitles for the non-speaker events e.g. the location of the Buffet Lunch, party etc.

Reply:
Yes, that could be a future improvement.